package.json

```diff
@@ -167,9 +167,9 @@
     "hardhat": "^2.11.2",
     "husky": "^4.3.8",
     "i": "^0.3.7",
-    "ipfs": "^0.58.3",
+    "ipfs": "^0.66.0",
```

Comment:
this version of ipfs depends on @libp2p/prometheus-metrics@1.1.5, which requires node 18